api/src/main/java/io/strimzi/api/kafka/model/CruiseControlSpec.java

```diff
@@ -22,20 +24,37 @@
 )
 @JsonInclude(JsonInclude.Include.NON_NULL)
 @JsonPropertyOrder({
-        "replicas", "image", "tlsSidecar"})
+        "replicas", "image", "storage",
+        "listeners", "config",
+        "livenessProbe", "readinessProbe",
+        "jvmOptions", "resources",
+        "metrics", "logging", "tlsSidecar", "template"})
```

Comment:
we don't have "listeners" and "storage" for sure. Please sync with declared and used fields.

Reply:
I'll strip this out